File: level2/행렬의_곱셈.cpp
#include <string>
#include <vector>

using namespace std;

vector<vector<int>> solution(vector<vector<int>> arr1, vector<vector<int>> arr2) {
    vector<vector<int>> answer;
    int sum;

    for(int i=0; i<arr1.size(); i++ ){
        vector<int> ans;
        for(int j=0; j<arr2[0].size(); j++){
            sum = 0;
            for(int k=0; k<arr2.size(); k++)
                sum += arr1[i][k]*arr2[k][j];
            ans.push_back(sum);
        }
        answer.push_back(ans);
    }
    return answer;
}

File: JavaScript/level2/행렬의_곱셈.js
function solution(arr1, arr2) {
    var answer = [];
    
    for(let i=0; i<arr1.length; i++){
        let temp = [];
        for(let k=0; k<arr2[0].length; k++){
            let val = 0;
            for(let j=0; j<arr2.length; j++)
                val += (arr1[i][j] * arr2[j][k]);
            temp.push(val);
        }
        answer.push(temp);
    }
    
    return answer;
}